admin-dev/themes/new-theme/package.json

```diff
@@ -5,80 +5,90 @@
   "private": "true",
   "scripts": {
     "test": "./node_modules/.bin/mocha  --require babel-register --reporter spec \"tests/**/*.spec.js\"",
-    "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
-    "dev": "webpack --progress --colors --debug --display-chunks --watch",
-    "watch": "npm run dev"
+    "build": "webpack --progress --colors --debug --display-chunks --mode=production",
```

Comment:
Maybe we can drop the `--debug` and `--display-chunks` in prod mode now that we're at it, don't you think? It might accelerate things and reduce the noise for people who just want to compile assets. If people want to debug, then they should use the dev mode.